test/service/BUILD

```diff
@@ -21,5 +22,4 @@ cc_test(
         "//test/filters/oidc:mocks",
         "@com_google_googletest//:gtest_main",
     ],
-    linkstatic = select({"@boost//:osx": True, "//conditions:default": False}), # workaround for not being able to figure out how to link dynamically on MacOS
```

Comment:
does this mean we will not be able to run the test in Mac OSX? which is fine to me, just want to ocnfirm.

Reply:
We should `linkstatic=True` on macOS. This change includes that to be True anytime. So we don't have to be concern.